### PR TITLE
Fixes broken drilldown refresh

### PR DESCRIPTION
**Pull request by djsauble**
Fixes broken refresh behavior. Many of the views were refreshing some, but not all, of their stores (noted by @adreghiciu).

Before: http://take.ms/Xv2qS
After: http://take.ms/yb0Gv

### CODE REVIEW

**Comment by kellyrob99** [comment]
This seems to be a fairly common concern and should likely be shared code rather than impl'd separately for each controller.
Should be sufficient to iterate over the 'stores' config of the controller to load in response to a refresh event, and have child classes with additional/different behaviour override loadStores or trigger additional functions on refresh.

**Comment by djsauble** [comment]
That was @adreghiciu’s observation as well, and I agree.

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Browse.js [hunk 1]
```diff
@@ -166,6 +171,16 @@ Ext.define('NX.coreui.controller.Browse', {
   },
 
   /**
+   * @overload
```
Comment (kellyrob99) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Browse.js]: no such tag, I think you mean @override but it also looks like we might be mis-using that... https://github.com/senchalabs/jsduck/wiki/@override

**Comment by kellyrob99** [comment]
+1 provided no regressions to existing FTs

**Comment by djsauble** [comment]
http://bamboo.s/browse/NX3-OSSF551-3

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Browse.js [hunk 2]
```diff
@@ -166,6 +171,16 @@ Ext.define('NX.coreui.controller.Browse', {
   },
 
   /**
+   * @override
+   * Load all of the stores associated with this controller
+   */
+  loadStores: function() {
+    var me = this;
```
Comment (mcculls) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Browse.js]: IIRC single uses of the `me` variable were to be avoided in place of using `this` directly? ie. replace this section with `this.getStore('Repository').load();`

**Comment by mcculls** [comment]
+1 with minor comment

Also `fix-broken-drilldown-refresh` would have been a more descriptive branch name :)

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Search.js [hunk 3]
```diff
@@ -415,10 +415,10 @@ Ext.define('NX.coreui.controller.Search', {
   },
 
   /**
-   * @private
+   * @overload
```
Comment (mcculls) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Search.js]: Should this be `@override`?

**Pull request closed without merging**